.github/workflows/deploy.yml

```diff
@@ -0,0 +1,43 @@
+#
+# Build and deploy the website
+#
+name: Deploy
+
+# Run on main branch only
+on:
+  pull_request: # enable pull_request for testing
+  push:
+    branches:
+      - master
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Cancel Previous Runs
+        uses: styfle/cancel-workflow-action@0.9.0
+
+      - name: Checkout
+        uses: actions/checkout@v2
```

Comment:
I believe you need to change the settings of the `checkout` step. 

For old Academic themes, we have to include the theme as a submodule. A simple clone results in an empty folder for `themes/academic`.